diagnostic.md

```diff
@@ -21,13 +21,18 @@ Write the `curl` request you'd use to retrieve a list of all donuts on the
 server.
 
 ```sh
-# your answer here
+curl --include --request GET 'http://www.example.com/donuts/'
 ```
 
 Write an AJAX request to retrieve a list of all donuts on the server.
 
 ```js
-let getDonuts = /* your answer here */;
+let getDonuts = function (event){
+  event.preventDefault();
+  libraryApi.index()
+    .done(ui.logResponseBody)
+    .fail(ui.logRequestError);
+};
```

Comment:
1/2.  I see what you wanted to do here but this is not an ajax request, the libraryApu.index() function that this calls would be the ajax request